### PR TITLE
feat(rad-reporting): RAD Reporting

### PR DESCRIPTION
**Pull request by abhi-markan**
## Introduction
An enhancement to an existing report whereby the reporting script will read the data from the database and generate a CSV report in a bespoke format.

## Resolution
* Introduction of `database` and `io` helper functions.
* Introduction of constants.
* Addition of `report` directory in `.gitignore`.

### CODE REVIEW

File: utils/data-migration/helpers/database.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  try {
+    const { _id } = updatedDeal;
+    delete updatedDeal._id;
+
+    if (!connection) await connect();
+
+    const response = await connection.collection(CONSTANTS.DATABASE.TABLES.TFM_DEAL).updateOne(
+      { _id: { $eq: ObjectId(String(_id)) } },
+      { $set: updatedDeal },
+      { returnOriginal: false },
```
Comment (IgnatG) [inline utils/data-migration/helpers/database.js:142]: can we use here `{ returnDocument: 'after', returnNewDocument: true }`?
`returnOriginal` is deprecated

**Review by IgnatG — APPROVED** [review]
LGTM! - I've added a small comment. not a show stopper though

File: utils/data-migration/helpers/database.js (hunk 1 above)
Comment (abhi-markan) [reply utils/data-migration/helpers/database.js]: Have added.

**Review by IgnatG — APPROVED** [review]
LGTM

**Pull request merged**